docs/quickstart.md

```diff
@@ -0,0 +1,70 @@
+# Quickstart
+
+## Run bblfsh and gitbase Dependencies
+
+It is recommended to read about `bblfsh` and `gitbase` from its own documentation, but here is a small guide about how to run both easily:
+
+Launch [bblfshd](https://github.com/bblfsh/bblfshd) and install the drivers. More info in the [bblfshd documentation](https://doc.bblf.sh/user/getting-started.html):
+
+```bash
+$ BBLFSHD_CONTAINER=bblfsh
+$ BBLFSHD_IMAGE=bblfsh/bblfshd
+$ docker run --privileged
+    --publish 9432:9432
+    --volume /var/lib/bblfshd:/var/lib/bblfshd
+    --name ${BBLFSHD_CONTAINER}
+    ${BBLFSHD_IMAGE}
+$ docker exec -it ${BBLFSHD_CONTAINER}
+    bblfshctl driver install --recommended
+```
+
+[gitbase](https://github.com/src-d/gitbase) will serve git repositories, so it is needed to populate a directory with them:
+
+```bash
+$ REPOS_PATH=~/gitbase/repos
+$ mkdir -p ${REPOS_PATH}
+$ git clone git@github.com:src-d/go-git-fixtures.git ${REPOS_PATH}/go-git-fixtures
+```
+
+Install and run [gitbase](https://github.com/src-d/gitbase):
+
+```bash
+$ REPOS_PATH=~/gitbase/repos
+$ GITBASE_CONTAINER=gitbase
+$ BBLFSHD_CONTAINER=bblfsh
+$ GITBASE_IMAGE=dpordomingo/gitbase:latest
+$ docker run
+    --publish 3306:3306
+    --link ${BBLFSHD_CONTAINER}
+    --volume ${REPOS_PATH}:/opt/repos
+    --env BBLFSH_ENDPOINT=${BBLFSHD_CONTAINER}:9432
+    --name ${GITBASE_CONTAINER}
+    ${GITBASE_IMAGE}
+```
+
+
+## Run gitbase-playground
+
+Once bblfsh and gitbase are running and accessible, you can serve the playground:
+
+```bash
+$ GITBASE_CONTAINER=gitbase
+$ PLAYGROUND_CONTAINER=gitbasePlayground
+$ PLAYGROUND_IMAGE=src-d/gitbase-playground:latest
```

Comment:
In all 3 quickstart examples, what is the value having user to read and copypaste named ENV vars?

It's 9 extra LoC in docs and quickstart usually is shorter the better.

Reply:
For me using "variables" is more clear, because it exposes what things are in common between different steps, and the meaning in behind some names.
But if you prefer to provide only a "run this command, and go", instead of a kind of "tutorial", just confirm it, and I'll do the changes :handshake:

Reply:
Cool! Yes, I belive `"run this command, and go"` would fit best for quickstart.

Ant thank you for being receptive and open-minded about such feedback! 💪

Reply:
thanks for suggesting things :dancer: 
done by https://github.com/src-d/gitbase-playground/pull/15/commits/fcb3973186c8a5f0bd201006c7e1cc171772e5bd